libpod/container_internal.go

```diff
@@ -514,7 +514,7 @@ func (c *Container) setupStorage(ctx context.Context) error {
 	}
 
 	// only reconfig IDMappings if layer was mounted from storage
-	// if its a external overlay do not reset IDmappings
+	// if it is an external overlay do not reset IDmappings
```

Comment:
Slight preference for punctuating and capitalizing both lines.

Reply:
now fixed